### PR TITLE
[MNOE-850] Free pricing

### PR DESCRIPTION
**Pull request by BrunoChauvet**
- Do not display empty price for free pricing

### CODE REVIEW

File: src/app/views/provisioning/confirm.html [hunk 1]
```diff
@@ -56,6 +56,9 @@ <h1 translate>mno_enterprise.templates.dashboard.provisioning.confirm.title</h1>
           <span ng-show="vm.subscription.product_pricing.pricing_type == 'payg'">
             {{ 'mno_enterprise.templates.dashboard.provisioning.order.pricing_type.payg' | translate }}
           </span>
+          <span ng-show="vm.subscription.product_pricing.pricing_type == 'free'">
+            {{ 'mno_enterprise.templates.dashboard.provisioning.order.pricing_type.free' | translate }}
```
Comment (ouranos) [inline src/app/views/provisioning/confirm.html]: Did you add this locale to mnoe yaml files too?

File: src/app/views/provisioning/summary.html [hunk 2]
```diff
@@ -51,6 +51,9 @@ <h2 translate>mno_enterprise.templates.dashboard.provisioning.summary.subscripti
           <span ng-show="vm.subscription.product_pricing.pricing_type == 'payg'">
             {{ 'mno_enterprise.templates.dashboard.provisioning.order.pricing_type.payg' | translate }}
           </span>
+          <span ng-show="vm.subscription.product_pricing.pricing_type == 'free'">
+            {{ 'mno_enterprise.templates.dashboard.provisioning.order.pricing_type.free' | translate }}
```
Comment (ouranos) [inline src/app/views/provisioning/summary.html]: Is this locale in the mnoe yaml files?

File: src/locales/en.json [hunk 3]
```diff
@@ -666,6 +666,7 @@
   "mno_enterprise.templates.dashboard.provisioning.order.price": "Price",
   "mno_enterprise.templates.dashboard.provisioning.order.next": "Next",
   "mno_enterprise.templates.dashboard.provisioning.order.pricing_type.payg": "Usage Based",
+  "mno_enterprise.templates.dashboard.provisioning.order.pricing_type.free": "Free",
```
Comment (ouranos) [inline src/locales/en.json]: Needs to be added in mnoe if not already done

File: src/app/views/provisioning/summary.html (hunk 2 above)
Comment (BrunoChauvet) [reply src/app/views/provisioning/summary.html]: Already done https://github.com/maestrano/mno-enterprise/pull/628/files

**Pull request merged**